### PR TITLE
#160 EA-1949 added isShownBy to serialization.

### PR DESCRIPTION
**Pull request by rgraf**
isShownBy should be added to serialized response

### CODE REVIEW

File: entity-utils/src/main/java/eu/europeana/entity/utils/jsonld/EuropeanaEntityLd.java [hunk 1]
```diff
@@ -66,6 +67,10 @@ public JsonLdResource setEntity(Entity entity) throws UnsupportedEntityTypeExcep
 		
 		//common administrative information
 		putAggregationProperty(entity, ldResource);
+		
+		//isShownBy
+		putMapOfStringProperty(WebEntityFields.IS_SHOWN_BY,
```
Comment (gsergiu) [inline entity-utils/src/main/java/eu/europeana/entity/utils/jsonld/EuropeanaEntityLd.java]: This is not a MapOfString property. Please implement similar to depiction

File: entity-utils/src/main/java/eu/europeana/entity/utils/jsonld/EuropeanaEntityLd.java [hunk 2]
```diff
@@ -75,6 +80,22 @@ public JsonLdResource setEntity(Entity entity) throws UnsupportedEntityTypeExcep
 		return ldResource;
 	}
 	
+	private Map<String,String> createIsShownByResource(BaseEntity entity) {
```
Comment (gsergiu) [inline entity-utils/src/main/java/eu/europeana/entity/utils/jsonld/EuropeanaEntityLd.java]: Change implementation similar to createWikimediaResource

Comment (rgraf) [reply entity-utils/src/main/java/eu/europeana/entity/utils/jsonld/EuropeanaEntityLd.java]: Changed as required

File: entity-utils/src/main/java/eu/europeana/entity/utils/jsonld/EuropeanaEntityLd.java (hunk 1 above)
Comment (rgraf) [reply entity-utils/src/main/java/eu/europeana/entity/utils/jsonld/EuropeanaEntityLd.java]: Changed similar to depiction

**Pull request merged**